torch_xla/csrc/ir.cpp

```diff
@@ -102,14 +102,18 @@ std::string Use::ToString() const {
   return ss.str();
 }
 
-const xla::Shape& Value::xla_shape() const { return node->xla_shape(index); }
-
-const xla::Shape& Value::xla_node_shape() const { return node->xla_shape(); }
+const xla::Shape& Value::xla_shape() const {
+  Node* casted = dynamic_cast<Node*>(node.get());
```

Comment:
Let's have a separate pr to rename all `torch_xla::Node` to `torch_xla::XlaNode` so it is not as confusing(given there is a `torch::lazy::Node` already)

Reply:
Sounds good, I'll follow-up with a separate PR.